core/scripts/chaincli/handler/handler.go

```diff
@@ -262,8 +264,16 @@ func (h *baseHandler) launchChainlinkNode(ctx context.Context, port int, contain
 		}
 	}
 
-	// Create DB container
 	postgresContainerName := fmt.Sprintf("%s-postgres", containerName)
+
+	// If force flag is on, we check and remove containers with the same name before creating new ones
+	if force {
+		if err = checkAndRemoveContainer(ctx, dockerClient, postgresContainerName); err != nil {
+			return "", nil, fmt.Errorf("failed to remove container: %s", err)
```

Comment:
The directive for printing an error is %w rather than %s, or, you can keep the %s but change the err to err.Error() 👍